src/services/hbs.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+import { AUTH_SERVICE_URL, AUTH_SERVICE_VERSION } from '../utils/hbsConfiguration'
+import { httpCall } from '../utils/httpProvider'
+import { useHoloStore } from 'src/stores'
+
+async function authCall(args) {
+  return httpCall({
+    serviceUrl: AUTH_SERVICE_URL,
+    version: AUTH_SERVICE_VERSION,
+    method: 'post',
+    ...args
+  })
+}
+  
+  export async function authenticateAgent(email, public_key) {
+    console.log(`hbs->authenticateAgent - email: ${email} public_key: ${public_key}`)
+
+    const holoStore = useHoloStore();
+    const payload = {
+      "email": email,
+      "timestamp": Date.now() - (30 * 1000), // Subtract 30 sec to prevent "future" timestamp error from API
```

Comment:
> // Subtract 30 sec to prevent "future"

If I remember correctly, the expiration is set for a 5min duration. Is the call itself taking longer than 5min to complete?  

If we're coming to close to the time boundary that we exceed it sometimes, I'd be in favor of increasing the time limit in hbs to support this, rather than alter the timestamp of the actual payload.  Thoughts?

Reply:
Per our earlier conversation I'm going to keep this until we understand why we have a slight clock disconnect.